src/views/Project/Details/Frameworks/FrameworkDetail/index.js

```diff
@@ -127,6 +130,59 @@ const projectRendererParams = (_, p) => ({
 
 const hideQuestionnaire = ({ accessQuestionnaire }) => !accessQuestionnaire;
 
+function saveAs(uri, filename) {
+    const link = document.createElement('a');
+    if (typeof link.download === 'string') {
+        link.href = uri;
+        link.download = filename;
+
+        document.body.appendChild(link);
+
+        link.click();
+
+        document.body.removeChild(link);
+    } else {
+        window.open(uri);
+    }
+}
```

Comment:
We should move this to utils

Reply:
this is very custom tailored to this particular case